uktrade/data-hub-frontend
# Remove add investment project button from UK companies

## Description of change

Remove add investment project button from UK companies

## Test instructions

Make sure the feature flag **company-hide-add-investment-button-for-uk** is set to true and for all companies with a ukregion, this should be hidden.

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/19926221/148251919-18063dc9-a414-44c6-ad3d-756417a9d03c.png)

### After

![image](https://user-images.githubusercontent.com/19926221/148250752-b6f4c90d-dc04-4371-9a94-55f1e2eb42c1.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
